plugins/pyenv/pyenv.plugin.zsh

```diff
@@ -18,7 +18,10 @@ for pyenvdir in "${pyenvdirs[@]}" ; do
         export PYENV_ROOT=$pyenvdir
         export PATH=${pyenvdir}/bin:$PATH
         eval "$(pyenv init - zsh)"
-        eval "$(pyenv virtualenv-init - zsh)"
+
+        if [[ "$(pyenv commands | grep virtualenv-init -c  > /dev/null 2>&1)" == `false` ]]; then
```

Comment:
A better alternative and common practice is using `grep`'s `-q` (quiet) argument. The final version should be:
```zsh
if pyenv commands | grep -q virtualenv-init; then
```